modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java

```diff
@@ -97,6 +114,43 @@ private Map<String, String> _getMap(String fileName) throws Exception {
 		return map;
 	}
 
+	private String _replaceImpl(
```

Comment:
`_replaceImpl()` is a somewhat ambiguous name (what is being "implemented"?). I think you can just name this `_replaceClassNames()`.